### ISSUE TITLE
loss doesn't decrease

### ISSUE THREAD

**Issue by Haerxu** [pengsongyou/openscene#46]
Hi,

I want to apply openscene in my own dataset. I follow the data preprocess instruction. However, the distillation loss always fluctuate around 0.2, like this:
![image](https://github.com/pengsongyou/openscene/assets/142866313/790e8f7b-a3fa-4966-aeb9-00e9b3a38ddf)

And the validation iou is around 0:
![image](https://github.com/pengsongyou/openscene/assets/142866313/278eac6b-1a96-41c8-9bf3-ba7caa38526b)

The distillation training script is adapted from scannet. For hyper-parameter setting, I only changed batch size from 8 (original in scannet) to 4 in order to fit the gpu memory. I suspect it is because of the different batch size since the learning rate is also related to batch size:
![image](https://github.com/pengsongyou/openscene/assets/142866313/6a25e5e6-3a44-4609-a576-9a775780da9e)

Do you know how to solve this problem?

-----------------------
Also, I test the performance by using 2d features to represent point cloud. The results are okay (miou=0.2391).

**Comment by pengsongyou** [comment]
Emmmm that is strange. Could you try training with just 1 scene instead of the entire dataset, and see if your network is able to overfit this scene (i.e., getting similar results to 2D fusion results like miou=0.2391)?

**Comment by Haerxu** [comment]
Thanks for the reply. I found a bug in my 3d data processing code. It can produce reasonable performance now.

By the way, does batch size affect the performance a lot? When I reproduce the results for Scannet, I set the batch_size=4 to fit in the 24GB GPU. But the miou of 3D distillation model is around 42 (46 in the table 4 of paper).

**Comment by pengsongyou** [comment]
Sorry for the late reply, yes, batch_size makes some differences, so maybe you can consider using multi-GPU training to increase the batch_size?

Best
Songyou
